flink-runtime/src/main/java/org/apache/flink/runtime/io/disk/iomanager/AsynchronousFileIOChannel.java

```diff
@@ -28,6 +28,7 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import static org.apache.flink.util.FileUtils.writeCompletely;
```

Comment:
i would avoid import static

Reply:
I would slightly lean towards avoiding it as well. But I won't argue about this.